juju-tutorials/how-to-deploy-CDK-on-openstack.md

```diff
@@ -0,0 +1,137 @@
+[comment]: <> (How to deploy CDK on OpenStack)
+
+## Introduction
+
+Duration: 20:00
+
+This tutorial provides information on how to deploy a [CDK][charmed-distribution-kubernetes] environment on
+the [OpenStack-base][openstack-base] environment.
+
+## Requirements
+
+There are two prerequisites you will need:
+
+1. OpenStack is configured with valid credentials (e.g. `source novarc`, represent the OpenStack credential, belonging
+to OpenStack on top of which you are deploying [OpenStack-base][openstack-base] and if you don't have it, ask your
+OpenStack administrator)
+1. Juju installed (see [Installing Juju][juju-install]), and the OpenStack environment is configured as cloud provider
+
+### Kubernetes
```

Comment:
I think this section about kubernetes should not be included because it already exists at [1]. I think the tutorial that points to this "how to deploy cdk on openstack" tutorial should actually point to Martin's tutorial (and this file be removed?)

1. https://discourse.charmhub.io/t/how-to-perform-maintenance-on-a-kubernetes-worker/3910